scripts/gc_store_site_inventory.py

```
@@ -471,6 +476,653 @@ def scan_file(path: Path) -> list[Finding]:
+        errors.extend(runtime_errors)
+        summary["runtime"] = runtime_summary
+    except VerificationRot as exc:
+        rot.append(str(exc))
+    return errors, rot, summary
+
+
+def unverified_marker_counts(files: dict[str, str]) -> dict[str, int]:
+    """Markers whose class remains human-audited only — declared, not hidden."""
+    counts: dict[str, int] = {}
+    unverified_re = re.compile(r"GC_STORE_AUDIT\((ROOT|INIT|POINTER_FREE|STACK)\)")
+    for rel, text in files.items():
```

Ruff (0.16.1)

[warning] Loop control variable `rel` not used within loop body

Rename unused `rel` to `_rel`

(B007)